instagram/src/components/PostContainer/PostContainer.js

```diff
@@ -0,0 +1,22 @@
+import React from "react";
+import Post from "../Post/Post";
+import PropTypes from "prop-types";
+
+const PostContainer = ({ posts }) => {
+  return (
+    <section className="post-container">
+      <div className="inner-post-container">
+        {posts.map(post => (
+          <Post key={post.timestamp} post={post} />
+        ))}
+      </div>
+    </section>
```

Comment:
without another sibling element to "inner-post-container", do you really need both `section` and `div` here?